src/Console/ConfigurationResolver.php

```diff
@@ -775,7 +790,7 @@ function ($path) {
         }
 
         $nestedFinder = null;
-        $currentFinder = $this->iterableToTraversable($this->getConfig()->getFinder());
+        $currentFinder = $this->iterableToTraversable($currentFinder);
```

Comment:
This is the only place where `iterableToTraversable` is called now. I think we should drop it as it is private and trivial.